docs/source/advanced_topics/15_embedded_kvs.rst

```diff
@@ -0,0 +1,129 @@
+Embedded Key-Value Store
+--------------------------
+
+Savant pipeline provides and embedded key-value store available through:
```

Comment:
`Savant pipeline provides and embedded` => `Savant pipeline provides an embedded`